packages/fast-tooling-react/src/form/form-control-switch.tsx

```diff
@@ -76,8 +111,8 @@ class FormControlSwitch extends React.Component<FormControlSwitchProps, {}> {
     /**
      * Renders the array form item
      */
-    private renderArray(): React.ReactNode {
-        const standardControl: StandardControlPlugin = this.props.controls.array;
+    private renderArray(control: StandardControlPlugin): React.ReactNode {
+        const standardControl: StandardControlPlugin = control;
```

Comment:
Whats the purpose of re-assigning this argument?

Reply:
I forgot to remove it :)